src/controllers/update/overseas.entity.change.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { saveAndContinue } from "../../utils/save.and.continue";
+import { Session } from "@companieshouse/node-session-handler";
+import { NoChangeKey } from "../../model/update.type.model";
+
+export const get = (req: Request, resp: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return resp.render(config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE, {
+      backLinkUrl: config.OVERSEAS_ENTITY_PRESENTER_URL,
+      templateName: config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL,
+      ...appData,
+    });
+  } catch (error){
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = async (req: Request, resp: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const session = req.session as Session;
+    let redirectUrl: string;
+    const appData: ApplicationData = getApplicationData(req.session);
+    const noChangeStatement = req.body[NoChangeKey];
+
+    if (appData.update){
+      appData.update.no_change = noChangeStatement;
```

Comment:
Originally string but boolean as the api change for saveAndResume expects a boolean

Reply:
Ok, so do we need to convert `noChangeStatement` to a boolean?

Reply:
I wouldn't expect a conversion. logging out extra data value is 1 (for yes) and 0(for no). Setting noChangeKey in mock test also maps to "1". 

Just looking at how if possible to see the exact value from test for setExtraData

Reply:
I was thinking that `appData.update.no_change` needed to be set to true or false and not 1 or 0, but as long as this is understood for when we read this value later on. Maybe it's best being a string rather than a boolean.

Reply:
@n-r-b changed it to string

Reply:
fixed

Reply:
Fixed